docs/en/api/POSIX-API.md

```diff
@@ -219,13 +219,13 @@ value of `-o`.
 The `-o [mount options]` must follow the `mount` command.
 
 Different versions of `libfuse` and `osxfuse` may support different mount options.
-The available Linux mount options are listed [here](http://man7.org/linux/man-pages/man8/mount.fuse.8.html).
+The available Linux mount options are listed [here](http://man7.org/linux/man-pages/man8/mount.8.html).
```

Comment:
hey @LuQQiu, can you confirm this is the link we want for the fuse?

Reply:
I found two potential reference for mount option for fuse:
`https://man7.org/linux/man-pages/man8/mount.fuse3.8.html`
`https://man7.org/linux/man-pages/man8/mount.fuse3.8@@libfuse.html`

Reply:
I think they are the correct ones as they are fuse-related. I'm updating it now.